packages/contracts-bedrock/scripts/deploy/ChainAssertions.sol

```diff
@@ -567,23 +569,79 @@ library ChainAssertions {
         }
     }
 
-    /// @notice Asserts that the SuperchainConfig is setup correctly
-    function checkOPContractsManager(Types.ContractSet memory _contracts, bool _isProxy) internal view {
-        OPContractsManager opcm = OPContractsManager(_contracts.OPContractsManager);
-        console.log(
-            "Running chain assertions on the OPContractsManager %s at %s",
-            _isProxy ? "proxy" : "implementation",
-            address(opcm)
+    /// @notice Asserts that the OPContractsManager is setup correctly
+    function checkOPContractsManager(
+        Types.ContractSet memory _contracts,
+        OPContractsManager _opcm,
+        IMIPS _mips
+    )
+        internal
+        view
+    {
+        console.log("Running chain assertions on the OPContractsManager at %s", address(_opcm));
+        require(address(_opcm) != address(0), "CHECK-OPCM-10");
+
+        // immutables
```

Comment:
what is this comment saying?

Reply:
removing it

Reply:
https://github.com/ethereum-optimism/optimism/pull/13510/commits/d2c9b037bbe63f2ca32e4da7914ff0dd4e043590